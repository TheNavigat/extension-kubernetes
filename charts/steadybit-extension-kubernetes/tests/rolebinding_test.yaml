templates:
  - rolebinding.yaml
tests:
  - it: manifest should match snapshot
    set:
      role:
        create: true
      roleBinding:
        create: true
      clusterRole:
        create: false
      clusterRoleBinding:
          create: false
    asserts:
      - matchSnapshot: { }
