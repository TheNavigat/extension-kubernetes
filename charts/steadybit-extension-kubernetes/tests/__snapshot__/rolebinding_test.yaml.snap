manifest should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: RoleBinding
    metadata:
      labels: null
      name: steadybit-extension-kubernetes
      namespace: NAMESPACE
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: Role
      name: steadybit-extension-kubernetes
    subjects:
      - kind: ServiceAccount
        name: steadybit-extension-kubernetes
        namespace: NAMESPACE
